xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -69,7 +69,7 @@ extern "C"
    */
   struct INPUTSTREAM
   {
-    static const unsigned int MAX_INFO_COUNT = 30;
+    static const unsigned int MAX_PROPERTY_COUNT = 30;
```

Comment:
I think it make sense to remove them here and take it from `static const unsigned int STREAM_MAX_PROPERTY_COUNT = 30;` on `InputStreamConstants.h`.

And to change it future also the PVR to take from here.

So all places have the correct size.

Reply:
Good idea